justfile

```diff
@@ -80,7 +80,7 @@ run-regtest args="":
 
 run-local-android args="":
     #!/usr/bin/env bash
-    LOCAL_IP=$(ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1')
+    LOCAL_IP=$(ip -o route get to 1 | sed -n 's/.*src \([0-9.]\+\).*/\1/p')
```

Comment:
`ip` is unfortunately not installed on a mac :( 
If this works for you, maybe we do a check for the operating system and execute `ip` for `linux` and `ifconfig` for `mac`?

Reply:
E.g. with this it should be relatively easy: https://github.com/casey/just#enabling-and-disabling-recipes

Reply:
the simplest way to check in a bash script would be checking by calling `uname` from a script. on macs, it returns `Darwin`, so you can match that for mac branch.

Reply:
`just` has a function for this https://github.com/casey/just#system-information